diff --git a/src/stdlib/SDL_mslibc.c b/src/stdlib/SDL_mslibc.c
index 3b3e7e276..777a85829 100644
--- a/src/stdlib/SDL_mslibc.c
+++ b/src/stdlib/SDL_mslibc.c
@@ -35,7 +35,7 @@
 
 #ifndef __FLTUSED__
 #define __FLTUSED__
-__declspec(selectany) int _fltused = 1;
+__declspec(dllexport) int _fltused = 1;
 #endif
 
 /* The optimizer on Visual Studio 2005 and later generates memcpy() and memset() calls.
